core/src/main/java/tc/oc/pgm/controlpoint/ControlPointModule.java

```diff
@@ -82,7 +82,7 @@ public ControlPointModule parse(MapFactory factory, Logger logger, Document doc)
           XMLUtils.flattenElements(doc.getRootElement(), "control-points", "control-point")) {
         ControlPointDefinition definition =
             ControlPointParser.parseControlPoint(factory, elControlPoint, false, serialNumber);
-        factory.getFeatures().addFeature(elControlPoint, definition);
+        factory.getFeatures().add(elControlPoint, definition);
```

Comment:
Is there a reason for changing this to `add()` from `addFeature()`? 

Seems to just make the file changes include lots of one-liner tweaks.

Reply:
To conform to `ContextStore` method names

Reply:
keep `addFeature`, add a new method `add` that calls addFeature, or make addFeature use add

Reply:
Let's go with Pab's suggestion so we don't accidentally make breaking changes.

Reply:
We can remove it in a future release.

Reply:
Good points! PGM is also an API after all, I'll change it as Pablo suggested